content/applications/inventory_and_mrp/inventory/product_management/product_tracking/package.rst

```diff
@@ -187,3 +187,17 @@ following fields:
 .. seealso::
    :doc:`Using cluster packages
    <../../warehouses_storage/advanced_operations_warehouse/cluster_picking>`
+
+View packages
+=============
+
+To view all packages go to :menuselection:`Inventory app --> Products --> Packages`. By default,
```

Comment:
I'm assuming the need to activate the 'Packages' setting is already mentioned above, but figured it was worth mentioning here, just in case it wasn't 😄